agent-bootstrap/src/main/java/io/opentelemetry/auto/typedspan/DbCassandraSemanticConvention.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opentelemetry.auto.typedspan;
+
+import io.opentelemetry.trace.Span;
+
+public interface DbCassandraSemanticConvention {
+  void end();
+
+  Span getSpan();
+
+  /**
+   * Sets a value for db.system
+   *
+   * @param dbSystem An identifier for the database management system (DBMS) product being used. See
+   *     below for a list of well-known identifiers..
+   */
+  public DbCassandraSemanticConvention setDbSystem(String dbSystem);
+
+  /**
+   * Sets a value for db.connection_string
+   *
+   * @param dbConnectionString The connection string used to connect to the database..
+   *     <p>It is recommended to remove embedded credentials.
+   */
+  public DbCassandraSemanticConvention setDbConnectionString(String dbConnectionString);
+
+  /**
+   * Sets a value for db.user
+   *
+   * @param dbUser Username for accessing the database..
+   */
+  public DbCassandraSemanticConvention setDbUser(String dbUser);
+
+  /**
+   * Sets a value for db.mssql.instance_name
```

Comment:
If this is Cassandra convention, it cannot have `db.mssql*` or `db.jdbc.driver_classname`